eo-maven-plugin/src/test/java/org/eolang/maven/testapi/package-info.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
@volodya-lombrozo it's a single class `FakeMaven`. Why do we need a package for it?